data/validator/standard_string_test.go

```diff
@@ -510,6 +612,46 @@ var _ = Describe("StandardString", func() {
 				})
 			})
 
+			Context("NotExists", func() {
```

Comment:
Yes, but the NotExists or Empty contexts are within other contexts (where the value being tested is different), so it isn't just a matter of repeated tests. Maybe Shared Examples would work. I'll look into it.